wiki/markdown/statusCodes.md

```diff
@@ -7,9 +7,9 @@ Status codes are generated according to the below tables.
 
 Below is the status codes:
 
-| Status code | Result                        | Message                                                                             | Comment                                                                                                                                                                                                                                                                                                                                                                        |
-|-------------|-------------------------------|-------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 200         | SUCCESS                       | Event sent successfully                                                             | Is returned if the request is completed successfully.                                                                                                                                                                                                                                                                                                                          |
+| Status code | Result                        | Message                                                                             | Comment                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Why are these status codes called "internal"?

Reply:
Removed the "Internal"